.github/workflows/scorecard.yml

```
@@ -0,0 +1,54 @@
+name: Scorecard supply-chain security
+on:
+  schedule:
+    - cron: '0 5 * * 1'
+  push:
+    branches: [ "main" ]
```

```suggestion
    branches:
      - main
```
I prefer the list syntax, less conflict potential